### DISCUSSION TITLE
no support for multiple primary keys

### DISCUSSION THREAD

**Discussion by branrx** [pawelsalawa/sqlitestudio#4335] [Q&A]
hi, I've recently started using SQLite studio, and I've noticed that I'm only able to assign a single primary key per table.

**Comment by pawelsalawa** [comment]
Composite primary keys (for multiple columns) can be added in the bottom panel of structure tab.

**Reply by branrx** [reply]
thanks a lot, completely missed it.

**Reply by Unsxlved** [reply]
I'm also facing the same issue, but I cant seem to be able to make the select a second primary key for my tables. Could you elaborate a little more?

**Reply by pawelsalawa** [reply]
![image](https://user-images.githubusercontent.com/481846/193892186-fbbd49d9-243e-4700-b591-3e78225d7fda.png)

**Reply by Unsxlved** [reply]
Yeah i see that but once I already have a column in there with a primary key. It no longer lets me click that button

**Reply by pawelsalawa** [reply]
Because you cannot define 2 primary keys. You either define one inside a column (and that will be for a single column only), or you do not mark it inside a column, but instead you define it here, at the bottom.

**Reply by Unsxlved** [reply]
Oh I get it now

Thanks. Your a life saver
